bdb/llmeta.c

```diff
@@ -1967,6 +1967,159 @@ int bdb_del_file_versions(
     return bdb_chg_file_versions(bdb_state, input_trans, NULL, bdberr);
 }
 
+static int bdb_del_file_version(
```

Comment:
Consider using `kv_del` in `llmeta.c`. Perpare your key and pass to that routine so you don't have to repeat boilerplate for various CURD on llmeta. There also exists `kv_put` and friends for other ops.

Reply:
I thought about going that route; however, I thought it would be useful to build on the existing LLMETA abstraction layer by providing the "single delete" primitive, using the standard ("well-tested"?) boilerplate.

Reply:
Indeed. `kv_del` was written with that intention.

Reply:
Right.  But `kv_del` appears to be generic for any LLMETA key.  What I was after was something that did also encapsulate the "file version" key building and other related logic.